### PR TITLE
Added images for cross platform controls for fluent website

### PR DESCRIPTION
**Pull request by ejlayne**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

Added images for cross platform controls for fluent website, and fixed a couple of copy bugs and broken links that I found while testing. Also updated the code examples so that a consumer can just copy paste the example into a react native App.js file, and have the code run.

#### Focus areas to test

Cross platform control pages.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/fluentui/pull/13106)

### CODE REVIEW

File: apps/fabric-website/src/pages/Controls/ButtonPage/docs/cross/ButtonUsage.md [hunk 1]
```diff
@@ -1,13 +1,31 @@
...
+
+#### Primary Button (Windows)
+
+<img src="https://static2.sharepointonline.com/files/fabric-cdn-prod_20200511.001/fabric-website/images/controls/cross/Button/Primary_button_windows.PNG"/>
+
+#### Default Button (macOS)
+
+<img src="https://static2.sharepointonline.com/files/fabric-cdn-prod_20200511.001/fabric-website/images/controls/cross/Button/Default_button_macos.png"/>
+
+#### Default Button (macOS)
```
Comment (markavitale) [inline apps/fabric-website/src/pages/Controls/ButtonPage/docs/cross/ButtonUsage.md:17]: Nit: should this be Primary Button?

Comment (ejlayne) [reply apps/fabric-website/src/pages/Controls/ButtonPage/docs/cross/ButtonUsage.md:17]: Yes, it should be, good catch.

Comment (ejlayne) [reply apps/fabric-website/src/pages/Controls/ButtonPage/docs/cross/ButtonUsage.md]: Thanks, fixed in new commit.

File: apps/fabric-website/src/pages/Controls/ButtonPage/docs/cross/ButtonUsage.md [hunk 2]
```diff
@@ -1,13 +1,31 @@
...
+
+<img src="https://static2.sharepointonline.com/files/fabric-cdn-prod_20200511.001/fabric-website/images/controls/cross/Button/Default_button_macos.png"/>
+
+#### Primary Button (macOS)
+
+<img src="https://static2.sharepointonline.com/files/fabric-cdn-prod_20200511.001/fabric-website/images/controls/cross/Button/Primary_button_macos.png"/>
+
 #### Example usage (from [ButtonFocusTest.tsx](https://github.com/microsoft/fluentui-react-native/blob/master/apps/fluent-tester/src/RNTester/TestComponents/Button/ButtonFocusTest.tsx))
 
 ```
```
Comment (micahgodbolt) [inline apps/fabric-website/src/pages/Controls/ButtonPage/docs/cross/ButtonUsage.md:23]:
```suggestion
```jsx
```

File: apps/fabric-website/src/pages/Controls/LinkPage/docs/cross/LinkUsage.md [hunk 3]
```diff
@@ -1,24 +1,40 @@
 Fluent UI React Native Link has default styling based on the Fluent Design Language. To customize the Link's visuals, you should use the "compose" pattern to customize the tokens. For more information on the compose pattern, please see our [compose documentation](https://github.com/microsoft/fluentui-react-native/blob/master/packages/framework/foundation-compose/README.md).
 
+### Link example
+
+<img src="https://static2.sharepointonline.com/files/fabric-cdn-prod_20200511.001/fabric-website/images/controls/cross/Link/Link_windows.PNG"/>
+
 #### Example usage (from [LinkTest.tsx](https://github.com/microsoft/fluentui-react-native/blob/master/apps/fluent-tester/src/RNTester/TestComponents/Link/LinkTest.tsx))
 
 ```
```
Comment (micahgodbolt) [inline apps/fabric-website/src/pages/Controls/LinkPage/docs/cross/LinkUsage.md:9]:
```suggestion
```jsx
```

File: apps/fabric-website/src/pages/Controls/SeparatorPage/docs/cross/SeparatorUsage.md [hunk 4]
```diff
@@ -1,31 +1,50 @@
+### Link example
+
+<img src="https://static2.sharepointonline.com/files/fabric-cdn-prod_20200511.001/fabric-website/images/controls/cross/Separator/Separator_windows.PNG"/>
+
 #### Example usage (from [SeparatorTest.tsx](https://github.com/microsoft/fluentui-react-native/blob/master/apps/fluent-tester/src/RNTester/TestComponents/Separator/SeparatorTest.tsx))
 
 ```
```
Comment (micahgodbolt) [inline apps/fabric-website/src/pages/Controls/SeparatorPage/docs/cross/SeparatorUsage.md:7]:
```suggestion
```jsx
```

File: apps/fabric-website/src/pages/Controls/PersonaPage/docs/cross/PersonaUsage.md [hunk 5]
```diff
@@ -1,18 +1,19 @@
+### Persona in various sizes
+
+<img src="https://static2.sharepointonline.com/files/fabric-cdn-prod_20200511.001/fabric-website/images/controls/cross/Persona/Persona_ramp.PNG"/>
+
 #### Example usage (from [PersonaCoinTest.tsx](https://github.com/microsoft/fluentui-react-native/tree/master/apps/fluent-tester/src/RNTester/TestComponents/PersonaCoin))
 
 ```
```
Comment (micahgodbolt) [inline apps/fabric-website/src/pages/Controls/PersonaPage/docs/cross/PersonaUsage.md:7]:
```suggestion
```jsx
```

File: apps/fabric-website/src/pages/Controls/TextPage/docs/cross/TextUsage.md [hunk 6]
```diff
@@ -2,21 +2,30 @@ Fluent UI React Native Text has default styling based on the Fluent UI Design La
...
+<img src="https://static2.sharepointonline.com/files/fabric-cdn-prod_20200511.001/fabric-website/images/controls/cross/Text/Text_quickbrownfox_macos.PNG"/>
+
+### Text ramp example
+
+You can specify the `variant` prop to apply font styles to Text. Examples of `variant` values are shown below.
+<img src="https://static2.sharepointonline.com/files/fabric-cdn-prod_20200511.001/fabric-website/images/controls/cross/Text/FURN_windows_text_ramp.PNG"/>
+
 #### Example usage (from [TextTest.tsx](https://github.com/microsoft/fluentui-react-native/tree/master/apps/fluent-tester/src/RNTester/TestComponents/Text))
 
 ```
```
Comment (micahgodbolt) [inline apps/fabric-website/src/pages/Controls/TextPage/docs/cross/TextUsage.md:20]:
```suggestion
```jsx
```

**Pull request merged**